hack/common/Makefile

```diff
@@ -45,6 +46,12 @@ run-without-lifecycle-manager: kyma \
 	apply-serverless \
 	verify-serverless
 
+.PHONY: run-without-lm-on-cluster
+run-without-lm-on-cluster: ## Deploy module without the lifecycle-manager on existing cluster.
+run-without-lm-on-cluster: deploy-cluster \
```

Comment:
Duplicate

Reply:
it's not. This is the only way to add comments (that can be used in the `make --help`) and required targets at the same time

Reply:
Yes, if you want to break lines it's the only option